docs/application/native/guides/machine-learning/machine-learning-training.md

```diff
@@ -462,6 +511,35 @@ After training, the model must be destroyed with `ml_train_model_destroy()`.
 `ml_train_model_add_layer()`, `ml_train_set_optimizer()`, and `ml_train_set_dataset()` transfers ownership to the model.
 `layers`, `optimizers` and `dataset` that belongs to the `model` are also deleted.
 
+## Using trained model for the inference
+
+Trained model can be used to inference with [Machine Learning Inference API](machine-learning-inference.md).
+
+Please be sure that the ini points to the correct weight file.
```

Comment:
```suggestion
Ensure that the INI file path points to the correct weight file.

Reply:
clarified to `Ensure that the INI file contains the correct weight file in `save_path`.